spec/features/gardens/show_spec.rb

```diff
@@ -21,7 +28,15 @@
   # And I see that this list only includes plants that take less than 100 days to harvest 
   it "displays a list of the plants included in the gardens plots" do
     visit "/gardens/#{@garden.id}"
-    save_and_open_page
-    expect(page).to have_content(@plant2.name)
+    # save_and_open_page
+
+    expect(@garden.unique_plants).to eq([@plant2]) # this should be a model test
```

Comment:
Would have updated my model to have the same before(:each) block at the beginning and this test would have gone in there but didn't want to scramble things up since I was up against the clock